interpreter.js

```diff
@@ -2807,8 +2807,24 @@ Interpreter.prototype['stepCallExpression'] = function() {
   if (!state.doneExec_) {
     state.doneExec_ = true;
     var func = state.func_;
-    if (!func || !func.isObject) {
-      this.throwException(this.TYPE_ERROR, func + ' is not a function');
+    var isNotFunction = ' is not a function';
+    if (!func) {
+      var callee = node['callee'];
+      var components = state.components_;
+      var functionName;
+      if (callee.object) {
+        functionName = callee.object.name;
+        if (components && Array.isArray(components) && 1 <= components.length) {
+          functionName += '.' + components[1];
+        }
+      }
+      else {
```

Comment:
nit; `}` and `else {` should go on the same line

Reply:
code no longer exists due to refactor :)